libsrc/target/tvc/tvc_set_ink.c

```diff
@@ -0,0 +1,23 @@
+#include <tvc.h>
+#include <stdint.h>
+
+void tvc_set_ink(color_or_index c) {
+    uint8_t *memP;
+    int res = tvc_get_vmode();
+    switch (res) {
+        case 0:  // 2 color
+        case 1:  // 4 color
+            memP = (uint8_t *)0x0B4D;
```

Comment:
You can actually do this which generates slightly more compact code:
    
        *((uint8_t *)0x0b4d) = c.paletteIndex

Reply:
Done.